jekyll/_cci2/creating-orbs.md

```diff
@@ -21,122 +21,56 @@ After authoring your orb, you can publish it with a [semantically versioned]({{s
-
-#### integration-test_deploy
-{: #integration-testdeploy }
-
-[`integration-test_deploy`](https://github.com/CircleCI-Public/Orb-Project-Template/blob/0354adde8405564ee7fc77e21335090a080daebf/.circleci/config.yml#L78) is the second and last workflow to run in our orb development pipeline. This workflow is automatically triggered, via the API, at the end of the `test-pack` workflow. This new workflow has access to the unique [development version]({{site.baseurl}}/2.0/orb-concepts/#orb-versions-development-vs-production-vs-inline) of the orb that was produced by the `test-pack` workflow.
-
-This second stage of the pipeline runs the [integration tests]({{site.baseurl}}/2.0/testing-orbs/#integration-testing), testing the new orb's functionality that has just been added and published to the development version..
-
-After integration testing, and only on the default branch, the deployment job will run. [orb-tools/dev-promote-prod-from-commit-subject](https://circleci.com/developer/orbs/orb/circleci/orb-tools#commands-dev-promote-from-commit-subject) is responsible for taking the SHA specific development version of the orb, and promoting it to a semantically versioned public release.
-
-{% raw %}
-```yml
-      - orb-tools/dev-promote-prod-from-commit-subject:
-          orb-name: <namespace>/<orb-name>
-          context: <publishing-context>
-          add-pr-comment: false
-          fail-if-semver-not-indicated: true
-          publish-version-tag: false
-          requires:
-            - integration-test-1
-          filters:
-            branches:
-              only: <your default branch>
-```
-{% endraw %}
+To publish a new version of our orb, we must tag our release. A tag can be created and pushed manually, however we recommend using [GitHub.com's Releases feature](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release).
 
-The `fail-if-semver-not-indicated` parameter is set to true by default, ensuring any commits without the proper [semver tag](#issue-a-new-release) in the commit title, result in a failed build.
+Using GitHub's Releases feature will allow us to publish "Release Notes", which will function as a changelog for the new version of the orb.
 
-You can optionally enable additional features such as publishing a version tag back to GitHub, and automatically posting new versions to the pull request via comment.
+Follow the GitHub docs to create a new release.
 
-##### Publish version tag to GitHub
-{: #publish-version-tag-to-github }
+   1. When asked to select a tag, specify the new tag that will be published with this release. Double check the current orb version in the [Orb Registry](https://circleci.com/developer/orbs). If you are unsure of what type of change you are making, we will have an opportunity to correct this in a moment. Ensure your tag fits the format `vX.Y.Z`
```

Comment:
Are these supplementary steps to the GH docs we just asked them to follow, or a summary of those steps?

Reply:
@KyleTryon